src/components/common/modal/ModalShow.tsx

```
@@ -6,67 +6,32 @@ import {
-            <ListItem>
-              Amendments
-              <p>{t("ACCEPT_AND_TERMS_8")}</p>
-            </ListItem>
-            <ListItem>
-              Acceptance of Terms
-              <p>{t("ACCEPT_AND_TERMS_9")}</p>
-            </ListItem>
-          </OrderedList>
+        <ModalBody maxHeight="400px" overflowY="auto">
+          {/* Render HTML from the language file */}
+          <div dangerouslySetInnerHTML={{ __html: termsHtml }} />
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)